framework/src/model.cpp

```diff
@@ -451,6 +451,9 @@ namespace gvk
 			
 			// We've got bone weights. Proceed as planned.
 			for (decltype(n) i = 0; i < n; ++i) {
+				// sort the current vertex' <bone id, weight> pairs descending by weight (so we can take the four most important ones)
+				std::sort(vTempWeightsPerVertex[i].begin(), vTempWeightsPerVertex[i].end(), [](std::tuple<uint32_t, float> a, std::tuple<uint32_t, float> b) { return std::get<float>(a) > std::get<float>(b); });
```

Comment:
I thought, for some reason, that ASSIMP would already sort by weight decreasing. Does it not?

Reply:
Looks like it does not - I encountered a model where the `bone_indices_for_mesh` and `bone_weights_for_mesh` returned bones with very small weight, ignoring the more important bones.